templates/openwisp2/settings.py

```diff
@@ -352,8 +352,7 @@
             'formatter': 'verbose',
             'filename': os.path.join(BASE_DIR, 'log/openwisp2.log'),
             'maxBytes': 15728640,
-            'backupCount': 3,
-            'formatter': 'verbose'
```

Comment:
why this change here?

Reply:
`'formatter': 'verbose'` is repeated, it's not causing error so I thought it sneaked in with one of the recent PRs, I can't think of a reason why repeating this key would be useful.